sdk/include/reactos/dbgbitmap.h

```diff
@@ -0,0 +1,359 @@
+/*
+ * PROJECT:     ReactOS SDK
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Helper functions to debug RTL_BITMAP
+ * COPYRIGHT:   Copyright 2024 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#pragma once
+
+#include <rtltypes.h>
+#include <rtlfuncs.h>
+
+typedef struct _RTL_BITMAP_DBG
+{
+    union
+    {
+        struct
+        {
+            ULONG SizeOfBitMap;
+            PULONG Buffer;
+        };
+        RTL_BITMAP BitMap;
+    };
+    ULONG NumberOfSetBits;
+    ULONG BitmapHash;
+} RTL_BITMAP_DBG, *PRTL_BITMAP_DBG;
+
+static inline
+ULONG fnv1a_hash(ULONG *data, size_t length)
```

Comment:
I'm curious: What made you decide to go with the FNV(-1a) hash function instead of another hash function?

Reply:
It's simple.